full/src/test/java/apoc/ApocSplitTest.java

```diff
@@ -0,0 +1,631 @@
+        "apoc.convert.toString",
+        "apoc.convert.toList",
+        "apoc.convert.toBoolean",
+        "apoc.convert.toNode",
+        "apoc.convert.toRelationship",
+        "apoc.convert.toSet",
+        "apoc.convert.toIntList",
+        "apoc.convert.toStringList",
+        "apoc.convert.toBooleanList",
+        "apoc.convert.toNodeList",
+        "apoc.convert.toRelationshipList",
+        "apoc.convert.toInteger",
+        "apoc.convert.toFloat",
+        "apoc.agg.percentiles",
+        "apoc.agg.product",
+        "apoc.agg.graph",
+        "apoc.agg.maxItems",
+        "apoc.agg.minItems",
+        "apoc.agg.statistics",
+        "apoc.agg.median",
+        "apoc.agg.nth",
+        "apoc.agg.first",
+        "apoc.agg.last",
+        "apoc.agg.slice");
+
+    public static final List<String> FULL_PROCEDURES = List.of(
+        "apoc.metrics.list",
+        "apoc.metrics.storage",
+        "apoc.metrics.get",
+        "apoc.monitor.kernel",
+        "apoc.monitor.store",
+        "apoc.monitor.ids",
+        "apoc.monitor.tx",
+        "apoc.ttl.expire",
+        "apoc.ttl.expireIn",
+        "apoc.static.list",
+        "apoc.static.set",
+        "apoc.bolt.load",
+        "apoc.bolt.load.fromLocal",
+        "apoc.bolt.execute",
+        "apoc.config.list",
+        "apoc.config.map",
+        "apoc.redis.getSet",
+        "apoc.redis.get",
+        "apoc.redis.append",
+        "apoc.redis.incrby",
+        "apoc.redis.hdel",
+        "apoc.redis.hexists",
+        "apoc.redis.hget",
+        "apoc.redis.hincrby",
+        "apoc.redis.hgetall",
+        "apoc.redis.hset",
+        "apoc.redis.push",
+        "apoc.redis.pop",
+        "apoc.redis.lrange",
+        "apoc.redis.sadd",
+        "apoc.redis.scard",
+        "apoc.redis.spop",
+        "apoc.redis.smembers",
+        "apoc.redis.sunion",
+        "apoc.redis.zadd",
+        "apoc.redis.zcard",
+        "apoc.redis.zrangebyscore",
+        "apoc.redis.zrem",
+        "apoc.redis.eval",
+        "apoc.redis.copy",
+        "apoc.redis.exists",
+        "apoc.redis.pexpire",
+        "apoc.redis.persist",
+        "apoc.redis.pttl",
+        "apoc.redis.info",
+        "apoc.redis.configGet",
+        "apoc.redis.configSet",
+        "apoc.systemdb.export.metadata",
+        "apoc.systemdb.graph",
+        "apoc.systemdb.execute",
+        "apoc.algo.aStarWithPoint",
+        "apoc.get.nodes",
+        "apoc.get.rels",
+        "apoc.cypher.runFile",
+        "apoc.cypher.runFiles",
+        "apoc.cypher.runSchemaFile",
+        "apoc.cypher.runSchemaFiles",
+        "apoc.cypher.parallel",
+        "apoc.cypher.mapParallel",
+        "apoc.cypher.mapParallel2",
+        "apoc.cypher.parallel2",
+        "apoc.gephi.add",
+        "apoc.mongo.aggregate",
+        "apoc.mongo.count",
+        "apoc.mongo.find",
+        "apoc.mongo.insert",
+        "apoc.mongo.update",
+        "apoc.mongo.delete",
+        "apoc.mongodb.get.byObjectId",
+        //"apoc.load.xls",
```

Comment:
See [this](https://github.com/neo4j-contrib/neo4j-apoc-procedures/pull/3079/commits/53da905ca870041d26a60c966f3c3e7803ce164f#r933362761) comment